superbench/benchmarks/micro_benchmarks/cuda_memory_bw_performance.py

```diff
@@ -108,23 +71,16 @@ def _process_raw_result(self, cmd_idx, raw_output):
         self._result.add_raw_data('raw_output_' + self._args.mem_type[cmd_idx], raw_output)
 
         mem_bw = -1
-        metric = ''
         valid = True
         content = raw_output.splitlines()
         try:
-            for index, line in enumerate(content):
-                if 'H2D' in line:
-                    metric = 'H2D_Mem_BW'
-                elif 'D2H' in line:
-                    metric = 'D2H_Mem_BW'
-                elif 'D2D' in line:
-                    metric = 'D2D_Mem_BW'
-                else:
-                    continue
-                line = line.split(',')[1]
-                value = re.search(r'(\d+.\d+)', line)
-                if value:
-                    mem_bw = max(mem_bw, float(value.group(0)))
+            metric = self._metrics[self._mem_types.index(self._args.mem_type[cmd_idx])]
```

Comment:
Why change the logic of ` def _process_raw_result(self, cmd_idx, raw_output):`? it's not in the parent class.